### PR TITLE
Double quote issue

### PR DESCRIPTION
**Pull request by sourabhgrover**
- Fixed double quote issue before date below the even title, on home page
- Added Unique key in List Component to remove unique key prop warning

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -36,14 +37,14 @@ function App() {
           </div>
           <Section>
             <h4>Past Events</h4>
-            {pastEvents.length > 0 && pastEvents.map((e) => (
-                <List key={e.id} title={e.title} description={`"on ${e.date} by ${e.speaker}`} />
+            {pastEvents.length > 0 && pastEvents.map((e, i) => (
+              <List key={i} title={e.title} description={`on ${e.date} by ${e.speaker}`} />
```
Comment (sarabs3) [inline src/App.js:41]: Don't use index as a key.

File: src/App.js [hunk 2]
```diff
@@ -36,14 +37,14 @@ function App() {
...
             ))}
           </Section>
           <Section>
             <h4>Upcoming Events</h4>
-            {events.length > 0 && events.map((e) => (
-                <List key={e.id} title={e.title} description={`"on ${e.date} by ${e.speaker}`} />
+            {events.length > 0 && events.map((e, i) => (
+              <List key={i} title={e.title} description={`on ${e.date} by ${e.speaker}`} />
```
Comment (sarabs3) [inline src/App.js:47]: Don't use index as a key.

**Review by sarabs3 — CHANGES_REQUESTED** [review]
Thank you for the PR. Please review the comments

File: src/App.js (hunk 1 above)
Comment (sourabhgrover) [reply src/App.js:41]: Yeah my bad,
I will update my PR and  remove unique key logic from this PR

And will open a separate issue for Unique key Warning.

@sarabs3 Does this make sense ?

Comment (sarabs3) [reply src/App.js:41]: Sure. Thank you.
I noticed that API is not returning a unique identifier for the events. I just created a PR for that now.
API code is in gcp-api branch.

Comment (jdecode) [reply src/App.js:41]: @sarabs3  - Do we need to have unique identifiers for the events?

Comment (sarabs3) [reply src/App.js:41]: yes

**Comment by jdecode** [comment]
@sourabhgrover @sarabs3  - Is this good to go or are we still waiting for changes?

**Comment by sourabhgrover** [comment]
@jdecode , I think so i need to create a new PR for this as we have changed the Repo structure

**Comment by jrishabh55** [comment]
#closing the PR

**Comment by jdecode** [comment]
The issue is still there on the main site - devtalks.in
Screenshot from 2020-07-30 12-25-09

**Comment by jrishabh55** [comment]
@jdecode i just pushed a commit to solve it https://github.com/Devtalks-India/devtalks/commit/89e2d2268bdebd04e79551b4764c7e98ae6e5e00 the PR was inactive for so long that i thought its better to close it

**Comment by jrishabh55** [comment]
Screenshot 2020-07-30 at 12 43 15 PM
Build Deployed

**Comment by jrishabh55** [comment]
Closing the PR, Cause issue is already fixed.

**Pull request closed without merging**